clouds/aws/aws_clients.py

```diff
@@ -0,0 +1,18 @@
+import boto3
+from botocore.exceptions import BotoCoreError
+from simple_logger.logger import get_logger
+
+LOGGER = get_logger(name=__name__)
+
+
+class AWSClient:
+    def __init__(self, region="us-east-1", **kwargs):
+        """Default region is us-east-1 also when param region_name is not given"""
+        LOGGER.info(
+            f"Creating {kwargs.get('service_name') or 'AWS'} client using region {region}."
```

Comment:
already commented on this - `service_name` should be an arg

Reply:
Misunderstand - I want to have the default AWS client session creation, which **doesn't** get service_name argument.
[Reference -](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/boto3.html#:~:text=Create%20a%20low%2Dlevel%20service%20client%20by%20name%20using%20the%20default%20session.)
_Create a low-level service client by name using the default session._

Reply:
For a better description passing `service_name=None` returns with error:

`Unknown service: 'None'. Valid service names are: accessanalyzer, account, acm....`